python/tank/platform/engine.py

```diff
@@ -882,7 +933,7 @@ def _initialize_dark_look_and_feel(self):
             fh.close()
             
             # set the std selection bg color to be 'shotgun blue'
-            self._dark_palette.setBrush(QtGui.QPalette.Highlight, QtGui.QBrush(QtGui.QColor("#30A7E3")))
+            self._dark_palette.setBrush(QtGui.QPalette.Highlight, QtGui.QBrush(QtGui.QColor(constants.SG_HIGHLIGHT_COLOR)))
```

Comment:
Like mentioned above, switch to getting the color directly from the dictionary to avoid having to have a constant per standard style constant?